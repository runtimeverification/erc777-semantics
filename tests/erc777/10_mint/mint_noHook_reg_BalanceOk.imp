test.preamble();

// operatorMint() through default operator (unless otherwise specified), operatorMint() from caller.
// regular recipient address with no hook, minting new balance

test.makeCaller(5);
test.operatorMint(5, 1000, "", ""); //from caller

test.makeCaller(8);
test.operatorMint(6, 1000, "", ""); //through default operator
