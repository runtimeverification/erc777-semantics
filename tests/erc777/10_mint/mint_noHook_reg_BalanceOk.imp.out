Changing the caller to 5
5 is minting 1000 for 5 (data:""),  (operatorData:"")
Balance of 5 is 1050
Total supply is 1550
Changing the caller to 8
8 is minting 1000 for 6 (data:""),  (operatorData:"")
Balance of 6 is 1060
Total supply is 2550
